packages/airplane/internal/api/client.ts

```diff
@@ -87,10 +88,15 @@ export class Client {
     }>("/v0/sessions/get", { id: sessionID });
   }
 
-  async createPrompt(params: ParamSchema[]): Promise<string> {
+  async createPrompt(params: ParamSchema[], opts?: PromptOptions<unknown>): Promise<string> {
```

Comment:
Add a `CreatePromptRequest` type to `types.ts` with schema/reviewers/confirmText/... then use that here instead. `PromptOptions` includes fields that are not used here.

```suggestion
  async createPrompt(req: CreatePromptRequest): Promise<string> {
```